SECURITY.md

```diff
@@ -0,0 +1,24 @@
+# Security Policy
+
+## Supported Versions
+
+Supported versions include Red v3 and higher. Red v2 and lower are end of life. And will not receive security updates of any kind. If you are using v2 we recommend you upgrade to v3.
```

Comment:
For consistency with ``[p]help``'s credits, we should capitalise V2 and V3.
```suggestion
Supported versions include Red V3 and higher. Red V2 and lower are end of life. And will not receive security updates of any kind. If you are using V2 we recommend you upgrade to V3.
```

Reply:
I'm not worried about this one. Especially due the fact that this documentation is aimed at people with a technical understanding of security exploits and so forth. Having this parallel with ``help`` is not a big deal.